src/geyser/bin.zig

```diff
@@ -0,0 +1,278 @@
+const std = @import("std");
+const sig = @import("sig");
+const cli = @import("zig-cli");
+
+pub const Config = struct {
+    pipe_path: []const u8 = sig.VALIDATOR_DIR ++ "geyser.pipe",
+    measure_rate_secs: u64 = 5,
+    geyser_bincode_buf_len: u64 = 1 << 29,
+    geyser_io_buf_len: u64 = 1 << 29,
+    owner_accounts: [][]const u8 = &.{},
```

Comment:
Does this have to be a mutable slice? Is it a limitation of `zig-cli`?

Reply:
5b03899